### DISCUSSION TITLE
Issue : FlowDirection = "RightToLeft" does not show correct chat axis

### DISCUSSION THREAD

**Discussion by drivingtheproject** [Live-Charts/LiveCharts2#803] [Q&A]
Hi,

When I set **FlowDirection = "RightToLeft"** to Chart control , it does not work properly.  It just mirror the axis.

**`<Grid>
    <lvc:CartesianChart FlowDirection="RightToLeft" Series="{x:Bind Series, Mode=OneWay}" />
</Grid>`**

How to solve this?

Thanks,
Saurabh

**Output**

![image](https://user-images.githubusercontent.com/622709/205757941-13f9c7d9-1c91-4482-ad25-1ca7670d2ddf.png)

**Comment by beto-rodriguez** [comment]
Hello, you can try to set the `Axis`.`IsInverted` property to `true`. https://github.com/beto-rodriguez/LiveCharts2/issues/154

**Reply by drivingtheproject** [reply]
@beto-rodriguez , Thanks a lot for quick reply , I will have a look.

Cheers,
Saurabh
